packages/cli/src/util/index.js

```diff
@@ -28,6 +29,7 @@ export default class Now extends EventEmitter {
   }) {
     super();
 
+    this.url = url;
```

Comment:
How was this working before?? 🤔

Reply:
The entire `deployment` object was passed, I don't know exactly what property it took for this to work. What I did was create a new property for that case 👍